File: problemas/iniciante/1150/1150.jl
using Printf

x = parse(Int, readline())
z = x
while (z <= x)
  global z = parse(Int, readline())
end

soma = x
resposta = 1
while (soma <= z)
  global soma += x + resposta
  global resposta += 1
end

@printf "%d\n" resposta

File: problemas/iniciante/1150/1150.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let x: i32 = input.trim().parse().unwrap();

    let mut z: i32 = 0;
    while z <= x {
        input = String::new();
        io::stdin().read_line(&mut input);
        z = input.trim().parse().unwrap();
    }

    let mut soma: i32 = 0;
    let mut resposta: i32 = 0;
    while soma <= z {
        soma += x + resposta;
        resposta += 1;
    }
    println!("{}", resposta);
}
